src/toniarts/openkeeper/game/console/GameConsole.java

```diff
@@ -169,7 +169,10 @@ public void execute(final String[] args) {
                 case ADD_GOLD:
                     try {
                         int amount = Integer.parseInt(args[1]);
-                        stateManager.getState(WorldState.class).addGold(keeper.getId(), amount);
+                        stateManager.getState(GameClientState.class)
```

Comment:
This does nothing but corrupts the client state. The moneys are kept on the server side.

Reply:
throw NotImplementedException()? :D I mean, it would probably require its own interface for these debug methods. `GameClientService` does have `triggerCheat` already. Maybe something similar then for debug functions, at least they should be grouped together for easy identification and validation.